OpenRobertaRobot/src/main/java/de/fhg/iais/roberta/bean/CompilerSetupBean.java

```diff
@@ -40,6 +47,20 @@ public Builder setCompilerResourcesDir(String compilerResourcesDir) {
             return this;
         }
 
+        public Builder setCompilerBinaryName(PluginProperties properties) {
+            if ( SystemUtils.IS_OS_LINUX ) {
+                if ( System.getProperty("os.arch").contains("arm") ) {
+                    compilerWorkflowBean.compilerBinaryName = properties.getStringProperty("robot.plugin.compiler.executable.arm");
+                }
+                compilerWorkflowBean.compilerBinaryName = properties.getStringProperty("robot.plugin.compiler.executable.linux");
+            } else if ( SystemUtils.IS_OS_MAC ) {
+                compilerWorkflowBean.compilerBinaryName = properties.getStringProperty("robot.plugin.compiler.executable.mac");
+            }else{
+                compilerWorkflowBean.compilerBinaryName = properties.getStringProperty("robot.plugin.compiler.executable");
```

Comment:
Best add an explicit Windows branch + property, and throw a DbcException when it's not one of those.